parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -50,12 +50,16 @@ else
     REBOOTREQUIRED=false
 fi
 
-if [ -f /var/log.vhd/azure/golden-image-install.complete ]; then
+
+VHD_LOGS_FILEPATH=/opt/azure/vhd-install.complete
+if [[ "${IS_VHD}" = true ]] then
+    # TODO (09/13): uncomment when new VHD is published
+    # if [ -f $VHD_LOGS_FILEPATH ]; then
+    #     echo "Using VHD distro but file $VHD_LOGS_FILEPATH not found"
+    #     exit $ERR_VHD_FILE_NOT_FOUND
+    # fi
     echo "detected golden image pre-install"
     FULL_INSTALL_REQUIRED=false
-    rm -rf /home/packer
```

Comment:
this isn't needed in here anymore since the deprovision step is happening in the vhd build